### PR TITLE
input component tests

### LINKED ISSUE #493: Input Component Test
/components/Input

### PR DESCRIPTION
**Pull request by Garman18**
Screenshot 2023-09-27 204245

### CODE REVIEW

**Comment by shahdivyank** [comment]
merge conflict

**Comment by Garman18** [comment]
removed merge conflicts

**Review by rfairooz — COMMENTED** [review]
Add your issue

File: cypress/component/Input.cy.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import Input from "@/components/admin/Input";
```
Comment (rfairooz) [inline cypress/component/Input.cy.jsx]: your test is on the component/Input not under admin. Fix the import

File: cypress/component/Input.cy.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          placeholder="Enter your name"
+          value={user.name}
+          user={user}
+          setUser={setUser}
+        />
+      );
+    };
+
+    cy.mount(<Parent />);
+    cy.get("p").should("have.text", "Name");
```
Comment (rfairooz) [inline cypress/component/Input.cy.jsx]: look at reference code carefully. We are using "data-cy-" to grab this data specifically for cypress testing. You have to add this label in the main component/Input file. Again check out the reference code for the other input test.

**Review by rfairooz — COMMENTED** [review]
The test is not passing. Did you test this code before pushing? (you test with : npm run dev => then cypress should open up => go to component tests)

**Comment by Garman18** [comment]
Screenshot 2023-09-27 204245

**Review by rfairooz — APPROVED** [review]

File: cypress/component/Input.cy.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
+import Input from "@/components/Input";
+import { useState } from "react";
+import { USER } from "../../src/data/forms/User";
+
+describe("Input", () => {
+  it("Placeholder", () => {
```
Comment (shahdivyank) [inline cypress/component/Input.cy.jsx:6]: mount it within the parent and still do the user use state

**Review by rfairooz — APPROVED** [review]

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**